example_services/templating.toml

```diff
@@ -0,0 +1,20 @@
+command = "/bin/bash -c 'echo hello world {{ env('USER')}}'"
+start-delay = "0s"
+stdout = "STDOUT"
+stderr = "/tmp/{{ env('USER') }}.horust_example.error.log"
+user = "{{ env('USER') }}"
```

Comment:
This also needs to be updated

Reply:
Everything should be cleaned in the newest commit